infra-as-code/bicep/modules/hubNetworking/hubNetworking.bicep

```diff
@@ -74,9 +74,19 @@ param parAzFirewallTier string = 'Standard'
   '1'
   '2'
   '3'
+  '1,2,3'
 ])
 @description('Availability Zones to deploy the Azure Firewall across. Region must support Availability Zones to use. If it does not then leave empty.')
-param parAzFirewallAvailabilityZones array = []
+param parAzFirewallAvailabilityZones array 
+
+@allowed([
+  '1'
+  '2'
+  '3'
+  '1,2,3'
```

Comment:
I think that this is not needed as parameter type is array?

Reply:
@olljanat - I was unable either to deploy to multiple zones without adding it in explicitly. If you are able to deploy this module to multiple zones without explicitly defining it, then I will defer. If your PR covers greater breadth, then I can just remove this one in favor of yours. I will also be on the Bicep call on Thursday and we can discuss, if needed.

Reply:
You are correct, thanks for catching this. I removed the explicit 1,2,3 from the allowed values
@jtracey93 - This should be ready now